app/models/identity.rb

```diff
@@ -125,6 +125,10 @@ def full_name
     "#{first_name.try(:humanize)} #{last_name.try(:humanize)}".lstrip.rstrip
   end
 
```

Comment:
Instead of adding another method on identity, you can do:
 full_name.split(' ').reverse.join(', ')